.github/governance.yml

```diff
@@ -0,0 +1,62 @@
+version: v1
+
+issue:
+  labels:
+    - prefix: kind
+      list:
+        - bug
+        - enhancement
+        - tech-debt
+      multiple: false
+      needs:
+        comment: |
+          @$AUTHOR: There are no 'kind' label on this.
+
+          * `/kind bug`
+          * `/kind enhancement`
+          * `/kind tech-debt`
+
+    - prefix: component
+      list:
+        - component:cli
```

Comment:
they use `area` themselves,

that is easy to type, "component" is a long work to type, maybe "comp" or "area" better?

Reply:
area sounds better

Reply:
I'm keen to keep consistent with Argo CD, and I don't see Argo CD changing, so not sure what to do here.